03-create.md

```diff
@@ -6,7 +6,8 @@ minutes: 10
 ---
 > ## Learning Objectives {.objectives}
 > 
-> *   Create a local Git repository.
+> *   Learn how to create a local Git repository using `git init`.
+> *   Learn how to locate `.git` directory and understand what its purpose is. 
```

Comment:
This should be added, but could you write it as "Understand the purpose of the `.git` directory"